### DISCUSSION TITLE
What is the EXACT packet packet overhead?

### DISCUSSION THREAD

**Discussion by VA1DER** [wangyu-/udp2raw-tunnel#541] [Q&A]
I want to use udp2raw to help obfuscate a WireGuard connection.  I want my data to go out with exactly 1500 byte packets so the their size looks like everyone else's packet.  To calculate my exact MTU I need to know exactly how many bytes udp2raw adds as overhead.

Please put that information in the MTU in the wiki so people can perform the exact calculation and end up with precise packet sizes.

**Comment by wangyu-** [comment]
Hi, at the moment, I choose to not maintain this kind of info in wiki. Since that info is depended on you setting, for example `--auth-mode` `--cipher-mode` `--fix-gro` etc.    Also I cannot guarantee this value is not changed in newer udp2raw version.

To get the value, you can send a test packet via `nc`, then catch the output packet of udp2raw with tcpdump.

By the way, as a friendly remind `aes128cfb` and `xor`'s overhead is a fixed value. But aes128-cbc's is not, since padding to multiple of 16 is involved.
